sdk/include/aditof/frame_definitions.h

```diff
@@ -133,7 +133,7 @@ struct FrameDetails {
  * @struct Point3I
  * @brief Holds the xyz values of a frame
  */
-struct Point3I {
+struct Point3I_sdk {
```

Comment:
Why the need for this change?

Reply:
There was a conflict with the Point3D from the tofi_utils.h file. This is a temporary fix.